prov/efa/src/efa_errno.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2022 Amazon.com, Inc. or its affiliates. All rights reserved.
```

Comment:
Unrelated to the purpose of this PR. I noticed that our copyright notices in many files are out-of-date. Feel free to update those encountered in this PR.